test/configure.test.js

```diff
@@ -14,18 +14,10 @@ var configure = require('../lib/configure.js');
 exports.configureLogging = function (t) {
     var appName = 'muskie';
     var bunyanCfg1 = {
-        level: 'info',
-        syslog: {
-            facility: 'local0',
-            type: 'udp'
-        }
+        level: 'info'
     };
     var bunyanCfg2 = {
-        level: 'debug',
-        syslog: {
-            facility: 'local0',
-            type: 'udp'
-        }
+        level: 'debug'
     };
 
     var bunyanCfg3 = {
```

Comment:
Seems like `bunyanCfg1` and `bunyanCfg3` are redundant now? Likewise with `bunyanCfg2`/`bunyanCfg4`. It also seems like several of the tests are redundant?

Reply:
Yes, thanks for catching that. I'll update the tests.